scripts/build-catalogue.in

```diff
@@ -67,8 +67,8 @@ also <name>_gpu.cpp and <name>_gpu.cu (if not given -C).''')
                         help='Enable GPU support, valid options: cuda|hip|cuda-clang.')
 
     parser.add_argument('-C', '--no-cpu',
-                        action='store_false',
-                        default=True,
+                        action='store_true',
+                        default=False,
```

Comment:
It's redundant to set a default: https://docs.python.org/3/library/argparse.html#action

Reply:
Done, thanks for the catch.